.github/workflows/scheduled_test_cpu.yml

```diff
@@ -49,6 +49,17 @@ jobs:
       os: 'MacOS-latest'
       pytorch-dtype: ${{ matrix.pytorch-dtype }}
 
+  test-slow:
+    uses: ./.github/workflows/tests.yml
+    strategy:
+      matrix:
+        pytorch-dtype: ['float32', 'float64']
+    with:
+      python-version: '["3.10"]'
+      pytorch-version: '["2.0.1"]'
+      pytorch-dtype: ${{ matrix.pytorch-dtype }}
+      pytest-extra: '--runslow'
```

Comment:
maybe this should just be the option on `tests-cpu-ubuntu:` job to run all test suite into all torch supported versions